modules/kafka/kafka-dest-worker.c

```diff
@@ -57,32 +47,43 @@ _format_message_and_key(KafkaDestWorker *self, LogMessage *msg)
                         self->super.seq_num, NULL, self->key);
 }
 
-static rd_kafka_topic_t *
-_calculate_topic_from_template(KafkaDestWorker *self, LogMessage *msg)
+gchar *
```

Comment:
Please make this const, to show, the string is not needed to be freed by the caller.